package.json

```diff
@@ -74,9 +74,10 @@
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-config-react-app": "^7.0.1",
-    "eslint-plugin-cypress": "^2.15.1",
+    "eslint-plugin-cypress": "^2.15.2",
     "eslint-plugin-jsdoc": "^48.2.3",
     "eslint-plugin-react-redux": "^4.1.0",
+    "glob": "^10.3.12",
```

Comment:
`glob` can be removed before this goes in (after the find and replace script is removed)